lib/smart_answer/calculators/uk_visa_calculator.rb

```diff
@@ -106,6 +107,18 @@ def passing_through_uk_border_control?
       @passing_through_uk_border_control_answer == 'yes'
     end
 
+    def travelling_to_channel_islands_or_isle_of_man?
+      @travelling_to_cta_answer == 'channel_islands_or_isle_of_man'
+    end
+
+    def travelling_to_ireland?
+      @travelling_to_cta_answer == 'republic_of_ireland'
+    end
+
+    def travelling_to_elsewhere?
+      @travelling_to_cta_answer == 'somewhere_else'
+    end
+
```

Comment:
@leenagupte no.. I did not give any thought to the approach you have just mentioned.
I felt at the time that these predicate methods were sufficient enough.